src/Console/ClientCommand.php

```diff
@@ -126,7 +126,7 @@ protected function createClientCredentialsClient(ClientRepository $clients)
         } while (false === $allowedScopes = $this->parseAllowedScopes($allowedScopes));
 
         $client = $clients->create(
-            null, $name, '', false, false, true, $allowedScopes
+            null, $name, '', false, false, true, ! $this->option('public'), $allowedScopes
```

Comment:
@ravanscafi do you remember if this space between `!` and `$this` is correct on this repository?

Reply:
I can't remember, sorry